hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/compress/lz4/Lz4Compressor.java

```diff
@@ -76,6 +64,19 @@ public Lz4Compressor(int directBufferSize, boolean useLz4HC) {
     this.useLz4HC = useLz4HC;
     this.directBufferSize = directBufferSize;
 
+    try {
+      LZ4Factory lz4Factory = LZ4Factory.fastestInstance();
+      if (useLz4HC) {
```

Comment:
nit: seems we no longer need the field `useLz4HC` with this.

Reply:
Oh, yes.